petab/visualize/helper_functions.py

```diff
@@ -337,6 +349,101 @@ def get_default_vis_specs(exp_data,
     return vis_spec, exp_data
 
 
+def check_ex_visu_columns(vis_spec, dataset_id_list, legend_dict):
+    """
+    Check the columns in Visu_Spec file, if non-mandotory columns does not
+    exist, create default columns
+    """
+    if X_VALUES not in vis_spec.columns:
+        raise NotImplementedError(
+            "Please define column: \'xValues\' in visualization file.")
+    if Y_LABEL not in vis_spec.columns:
+        vis_spec.insert(loc=4, column=Y_LABEL, value='value')
+    if Y_VALUES not in vis_spec.columns:
+        vis_spec.insert(loc=4, column=Y_VALUES, value='')
+    if X_LABEL not in vis_spec.columns:
+        vis_spec.insert(loc=4, column=X_LABEL, value='time')
+    if X_OFFSET not in vis_spec.columns:
+        vis_spec.insert(loc=4, column=X_OFFSET, value=0)
+    if Y_SCALE not in vis_spec.columns:
+        vis_spec.insert(loc=4, column=Y_SCALE, value=LIN)
+    if X_SCALE not in vis_spec.columns:
+        vis_spec.insert(loc=4, column=X_SCALE, value=LIN)
+    if LEGEND_ENTRY not in vis_spec.columns:
+        if dataset_id_list is not [] and legend_dict is not {}:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Comparison to literal](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4981434)